src/main/java/com/austinv11/servicer/ServicerProcessor.java

```diff
@@ -94,12 +92,12 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
 
             try {
-                FileObject fo = filer.createResource(StandardLocation.CLASS_OUTPUT, "", serviceLocation);
+                FileObject fo = filer.createResource(StandardLocation.CLASS_OUTPUT, "", serviceLocation, s.getElements());
```

Comment:
What is the purpose of adding the elements at the end?

Reply:
This is an important part of incremental compilation because it tells the compiler everything that generated the specific resource. This allows Gradle to resend all of the specified elements if any of them change.

Reply:
Honestly, it is odd that google's implementation doesn't use it

Reply:
Sounds good